scan/lib/scan/options.rb

```diff
@@ -124,6 +124,11 @@ def self.available_options
                                      end),
 
         # other test options
+        FastlaneCore::ConfigItem.new(key: :testPlan,
```

Comment:
```suggestion
        FastlaneCore::ConfigItem.new(key: :testplan,
```

Any reason for the non common capitalization? For Ruby (how we use it) we would probably expect either `testplan` or `test_plan`.

Reply:
Ah, I see the Xcode option is like this. I think we should not use that pattern for fastlane code though.

Reply:
Thanks for the review :)

Yeah I took it initially from the Xcode parameter (even though `testplan` reads a little better to me). Definitely makes sense to prefer consistency in the Fastlane/ruby syntax instead.